### DISCUSSION TITLE
Database Design

### DISCUSSION THREAD

**Discussion by Mzeey** [mzeey-tutorials/NepaApp#9] [Announcements]
Hey, This is a discussion for the database design of our project, we will be discussing what SQL server we will use and how we will manage test data.

**Comment by devshalem** [comment]
Which SQL server are we to use

**Reply by Mzeey** [reply]
I don't know if we will use Microsoft SQL or SQLite

**Reply by Mzeey** [reply]
What do you think? 
Because sql lite we can easily carry the DB, around but with the MsSQL we will need to have a base dev sql dB, that we will raise an issue for before it is updated, and we will have to create test user accounts which we will document what they are and thier passwords

**Comment by Mzeey** [comment]
Okay so bro we are using Microsoft SQL. 
Now we need go structure the dB

**Reply by devshalem** [reply]
Ok noted

**Reply by devshalem** [reply]
Will begin the assigned task for Database Design
